core/src/test/java/io/confluent/kafka/schemaregistry/client/CachedSchemaRegistryClientTest.java

```diff
@@ -73,6 +73,23 @@ private Properties createConsumerProps() {
     return props;
   }
 
+  private Properties createCompositeConsumerProps() {
+    Properties props = new Properties();
+    props.put("bootstrap.servers", brokerList);
+    props.put("group.id", "avroGroup");
+    props.put("session.timeout.ms", "6000"); // default value of group.min.session.timeout.ms.
+    props.put("heartbeat.interval.ms", "2000");
+    props.put("auto.commit.interval.ms", "1000");
+    props.put("auto.offset.reset", "earliest");
+    props.put("key.deserializer", org.apache.kafka.common.serialization.StringDeserializer.class);
+    props.put("value.deserializer",
+        io.confluent.kafka.serializers.migration.CompositeDeserializer.class);
+    props.put("old.deserializer", io.confluent.kafka.serializers.KafkaAvroDeserializer.class);
+    props.put("confluent.deserializer", io.confluent.kafka.serializers.KafkaAvroDeserializer.class);
```

Comment:
does it make sense to keep these different?

Reply:
Are you referring to the property names?

Reply:
i meant using different deserializers

Reply:
Oh right, let me fix that